ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-ref.yaml

```diff
@@ -6,16 +6,18 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: cluster-secrets-openstack-vexxhost
+    mount_path: /var/run/cluster-secrets-openstack-vexxhost
+  - namespace: test-credentials
+    name: cluster-secrets-openstack-moc
+    mount_path: /var/run/cluster-secrets-openstack-moc
   env:
-    - name: CONFIG_TYPE
-    - name: CONFIG_API_VERSION
-    - name: OPENSTACK_EXTERNAL_NETWORK
-    - name: OPENSTACK_COMPUTE_FLAVOR
-    - name: OS_CLOUD
-    - name: BASE_DOMAIN
-    - name: KURYR_ENABLED
-      default: "false" 
+  - name: OS_CLIENT_CONFIG_FILE_VEXXHOST
+    default: '/var/run/cluster-secrets-openstack-vexxhost/clouds.yaml'
+  - name: OS_CLIENT_CONFIG_FILE_MOC
+    default: '/var/run/cluster-secrets-openstack-moc/clouds.yaml'
```

Comment:
can't we merge this two into one file?

Reply:
I see two problems with merging the secrets together:
- if the test container somehow gets compromises, that exposes the secret for all cloud platforms
- the clouds.yaml to environment variable conversion currently expects only one entry in the clouds.yaml file, see https://bugzilla.redhat.com/show_bug.cgi?id=1918680. Not that this is a big issue, since I'm going to make it more robust in https://issues.redhat.com/browse/OSASINFRA-2313.